pkg/controllers/jobset_controller_test.go

```diff
@@ -127,10 +128,26 @@ func TestIsJobFinished(t *testing.T) {
 
 func TestConstructJobsFromTemplate(t *testing.T) {
 	var (
-		jobSetName          = "test-jobset"
-		replicatedJobName   = "replicated-job"
-		jobName             = "test-job"
-		ns                  = "default"
+		jobSetName        = "test-jobset"
+		replicatedJobName = "replicated-job"
+		jobName           = "test-job"
+		ns                = "default"
+		jobAnnotations    = map[string]string{
+			"foo":  "bar",
+			"foo1": "bar",
+		}
+		jobLabels = map[string]string{
+			"foo":  "bar",
+			"foo1": "bar",
+		}
+		podAnnotations = map[string]string{
+			"hello":  "world",
+			"hello1": "world",
+		}
+		podLabels = map[string]string{
+			"hello":  "world",
```

Comment:
lets use different labels and annotations for job and pod

e.g., `pod-label: pod-key` etc.